### PR TITLE
[COMPLETED]DOCUMENTATION Update

### LINKED ISSUE #3: Add `DOCUMENTATION` on how to use this tool.
- [ ] Documentation on using the tool.
- [ ] Documentation for OSS formats.

### PR DESCRIPTION
**Pull request by RD-Tarun**
https://github.com/CSE-25/amrita_pyq/issues/3 - PR Issue Link

@Ashrockzzz2003 I've created the documentation for the CLI tool in OSS Format as well. Please review the changes and let me know if any further modifications are needed.

Thank You!!

### CODE REVIEW

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]
Please move the changes to `README.md` and delete `documentation.md`. That is where it's meant to be.

Add a point about the requirement of Amrita WiFi/VPN to access the tool in usage as a separate step.

**Comment by RD-Tarun** [comment]
alright!!

File: README.md [hunk 1]
```diff
@@ -1,11 +1,106 @@
...
+
+---
+
+
+### Step 5: View the Question Paper
+1. The question paper will automatically open as a **PDF** in your default web browser.
+
+### Output
+![image](https://github.com/user-attachments/assets/d926aad6-2adc-41ce-a1b4-c287f1ccab1d)
+![image](https://github.com/user-attachments/assets/62e4f6cb-2d38-41ec-9e00-2e1d347114e2)
```
Comment (Abhinav-ark) [inline README.md]: Can you replace this image with a screenshot of it opening in a browser?

**Review by Abhinav-ark — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RD-Tarun) [reply README.md]: Could you clarify whether you're referring to the image mentioned at Line 57 or Line 58, or both?

Comment (Abhinav-ark) [reply README.md]: 58

Comment (RD-Tarun) [reply README.md]: did the changes, please check it out

Comment (RD-Tarun) [reply README.md]: @Abhinav-ark

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]
So far looks good @RD-Tarun 
Consider adding `Amrita Logo`, `Go` logo, and possibly some relevant shield icons.

**Comment by RD-Tarun** [comment]
done, please check it out @Ashrockzzz2003 @Abhinav-ark

File: README.md [hunk 2]
```diff
@@ -1,11 +1,130 @@
-# amrita_pyq
+![Static Badge](https://img.shields.io/badge/ampyq-darkred?logoColor=white)
```
Comment (Ashrockzzz2003) [inline README.md:1]: Any reason for `ampyq`?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,130 @@
...
 
-We welcome contributions! Please see our [Contributing Guide](.github/CONTRIBUTING.md) for more details.
+# **Amrita PYQ CLI Tool Documentation**
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/44822d21-0e1d-4b3e-baa3-3450b5cc14fc" alt="amritalogo" width="550" height="200">
+</p>
 
-<!-- 
-## Code of Conduct
+## **Overview**
```
Comment (Ashrockzzz2003) [inline README.md]: When `##` makes it bold, why `**` again? Same below too

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]
Thanks for linking my portfolio instead of GitHub! : )

File: README.md (hunk 2 above)
Comment (RD-Tarun) [reply README.md:1]: image
"name is in the root.go file, and it sounds good as well, so why not keep it as the app name?" was my thought process

File: README.md (hunk 3 above)
Comment (RD-Tarun) [reply README.md]: To add significance to the heading, I can delete it if needed....

Comment (Ashrockzzz2003) [reply README.md]: I feel it's too much.

File: README.md (hunk 2 above)
Comment (Ashrockzzz2003) [reply README.md:1]: awsm

File: README.md (hunk 3 above)
Comment (RD-Tarun) [reply README.md]: okay then, will revert it

Edit : Done @Ashrockzzz2003 , check it out

File: README.md [hunk 4]
```diff
@@ -1,11 +1,130 @@
-# amrita_pyq
+![Static Badge](https://img.shields.io/badge/ampyq-darkred?logoColor=white)
+![Go Badge](https://img.shields.io/badge/Go-00ADD8?logo=go&logoColor=white)
+![Last Updated](https://img.shields.io/badge/Last%20Updated-December-00ADD8?style=flat&logoColor=white&labelColor=5E5D5D&color=D0A93E)
 
-## Contributing
 
-We welcome contributions! Please see our [Contributing Guide](.github/CONTRIBUTING.md) for more details.
+# Amrita PYQ CLI Tool Documentation
```
Comment (Ashrockzzz2003) [inline README.md]: The word `Documentation` is not necessary here ig.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,130 @@
...
+   - **Exit**: Press `0` to close the application.
+
+### Output
+![image](https://github.com/user-attachments/assets/cc4cdd7e-f1f4-4926-a00e-00adc098cb66)
+
+---
+
+# Local Development Environment Setup
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/7c2a4198-cfd9-4451-89a0-44637136e1f4" alt="golang" width="300" height="200">
```
Comment (Ashrockzzz2003) [inline README.md:84]: img too big!! 😆

**Review by Ashrockzzz2003 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (RD-Tarun) [reply README.md:84]: changes done @Ashrockzzz2003

**Review by Ashrockzzz2003 — APPROVED** [review]
LGTM

**Review by Abhinav-ark — APPROVED** [review]
LGTM.

**Pull request merged**